src/org/opendatakit/briefcase/ui/export/ExportPanelForm.java

```diff
@@ -120,6 +133,19 @@ private void setExporting(boolean active) {
     exporting = active;
   }
 
+  synchronized public void updateExportingLabel() {
+    String text = exportingLabel.getText();
```

Comment:
These six lines could be one by using ? :  .

Reply:
I think you're right, but I think the result could be harder to understand than in its current state...
There's something not quite right in checking against `EXPORTING_DOT_ETC` and then building everything else with magic strings... For example, if I change the value of `EXPORTING_DOT_ETC` and I forget to change that block, I break it.

I don't have a better solution right now, but I'm sure we will be coming back to this block...

Reply:
In other words: using a ternary doesn't help us come to a better solution, in my opinion

Reply:
Glad you’ll be coming back.

Reply:
As you have probably noticed, I love the *Conditional Operator* (the only ternary¹ operator).
https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.25

¹composed of three parts

Reply:
Conditional expressions are the best conditionals :)